tables/array.py

```diff
@@ -92,17 +82,10 @@ class Array(hdf5extension.Array, Leaf):
     # Class identifier.
     _c_classid = 'ARRAY'
 
-    # Lazy read-only attributes
-    # `````````````````````````
-    @lazyattr
```

Comment:
Is this wanted?

Reply:
No, sorry, a silly typo, thank you, fixed, repushed. I hoped I was able to remove plenty of comments and not break anything, so I let it directly to the CI. And CI and you saw it.